set_cost.py

```diff
@@ -10,14 +10,16 @@ def main():
   with open(CONFIG_FILE, 'r') as config_file_stream:
     config = yaml.safe_load(config_file_stream)
 
-  ec = ExpectedCosts(config)
+  shipment_info = ShipmentInfo(config)
   while True:
     order = input("Enter order ID: ").strip()
     if not order:
       break
     cost = float(input("Enter cost: ").strip())
-    ec.costs_dict[order] = cost
-    ec.flush()
+    # We don't (yet?) support manual entry of the email_id because it's a little
+    # bit of hassle to find through the Web UI.
+    shipment_info.shipments_dict[order] = OrderInfo(None, cost)
```

Comment:
we might want to reset the price on something for which we already have an email ID -- if it exists, we should use it

Reply:
Done.

Reply:
Let me know if this is correct now.